src/app/shared/view-issue/tester-response/tester-response.component.ts

```diff
@@ -139,7 +143,7 @@ export class TesterResponseComponent implements OnInit, OnChanges {
   }
 
   handleChangeOfDisagreeCheckbox(event, index: number) {
```

Comment:
```suggestion
  handleChangeOfRadioButton(event, index: number) {
```

We should rename usages of `checkbox` to `radio`

Reply:
Would `...DisagreeRadioButton` be better? Since `RadioButton` seems a little generic?

Reply:
Sure